App/Components/GroupCard/index.tsx

```diff
@@ -3,16 +3,16 @@ import { View, Text, TouchableOpacity } from 'react-native'
 import Icon from '@textile/react-native-icon'
 import TextileImage from '../TextileImage'
 import Avatar from '../Avatar'
-import { pb } from '@textile/react-native-sdk'
+import { IContact, IFiles } from '@textile/react-native-sdk'
 
 import styles, { cardImageStyle, ICON_WIDTH, ROW_COLUMN } from './statics/styles'
 import { color } from '../../styles'
 
 interface ScreenProps {
   id: string
   name: string
-  members: pb.IContact[]
-  thumb?: pb.IFiles
+  members: IContact[]
+  thumb?: IFiles
```

Comment:
Ok the removal of the `pb` namespace created tons of noise in this PR. I'll try to leave comments where anything actually interesting changed.